examples/src/main/java/ai/djl/examples/training/TrainPikachu.java

```diff
@@ -87,7 +87,7 @@ public static TrainingResult runExample(String[] args) throws IOException, Trans
             try (Trainer trainer = model.newTrainer(config)) {
                 trainer.setMetrics(new Metrics());
 
-                Shape inputShape = new Shape(arguments.getBatchSize(), 3, 256, 256);
+                Shape inputShape = new Shape(1, 3, 256, 256);
```

Comment:
Why?

Reply:
You probably shouldn't modify the example to use a fixed batch size of 1 rather than the variable version